rastervision_core/rastervision/core/rv_pipeline/rv_pipeline.py

```diff
@@ -127,6 +127,9 @@ def chip(self, split_ind: int = 0, num_splits: int = 1):
 
             def chip_scene(scene, split):
                 windows = self.get_train_windows(scene)
+                if len(windows) == 0:
+                    log.warn('Warning: scene produced no windows')
+                    return
```

Comment:
- I believe `log.warn()` is going to be deprecated in favor of `log.warning()`. 
- RV loggers [are configured to add](https://github.com/azavea/raster-vision/blob/14d21e9013b32bed7ce7215b857c17b61d2edf85/rastervision_pipeline/rastervision/pipeline/__init__.py#L33-L34) a "WARNING" prefix automatically, so `Warning:` here is redundant.
- I would additionally include the scene ID in the warning message.